src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        int range = 100;
```

Comment:
constant should be like this:
private static final int MAX_RANDOM_NUMBER = 100;